Update content on landing pages. (#51)
US-GHG-Center/ghgc-docs: services/jupyterhub.qmd

title: JupyterHub
---

The US GHG Center promotes the use of JupyterHub environments for interactive data science. JupyterHub enables you to analyze massive archives of Earth science data in the cloud in an interactive environment that alleviates the complexities of managing compute resources (virtual machines, roles and permissions, etc).

Users affiliated with the US GHG Center can get access to a dedicated JupyterHub service, provided in collaboration with [2i2c](https://2i2c.org/): [hub.ghg.center](https://hub.ghg.center/). Please find instructions for requesting access below.

If you are a scientist affiliated with other NASA projects such as [VEDA](https://nasa-impact.github.io/veda-docs/), [EIS](https://eis.smce.nasa.gov/about.html), and [MAAP](https://ops.maap-project.org), you can also keep using the resources provided by these projects. Through the use of open-source technology, we make sure our services are interoperable and exchangeable.

## To Get US GHG Center JupyterHub access:

>  The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our Hub Access Request form.following these steps:

Reviewer: ```suggestion
 The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://forms.gle/4Z7rkUkeK9MW4joi7).
```

* Make sure you have a [Github Account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). Take note of your [Github username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email).

> * Fill out the request form and provide needed information.

Reviewer: ```suggestion
* Fill out the [request form](https://forms.gle/4Z7rkUkeK9MW4joi7) and provide needed information.
```

* Watch your email for notification of authorization and the invite to join the [US GHG Center Hub Access Github Team](https://github.com/orgs/US-GHG-Center/teams/ghgc-hub-access).

> * Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your Github credentials.

Reviewer: ```suggestion
* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your GitHub credentials.
```

Reviewer: replace all Github -> GitHub

## To access User Notebooks

This site provides [Jupyter notebooks](../user_data_notebooks/casagfed-carbonflux-monthgrid-v3_User_Notebook.ipynb) showing how to load and analyze Earth data in the interactive cloud computing environment.

> Further instructions are included in each notebook.  

Reviewer: ```suggestion
Further instructions are included in each notebook. 

```

Users affiliated with the US GHG Center can get access to a dedicated JupyterHub service, provided in collaboration with [2i2c](https://2i2c.org/): [hub.ghg.center](https://hub.ghg.center/). Please find instructions for requesting access below.

> If you are a scientist affiliated with other NASA projects such as [VEDA](https://nasa-impact.GitHub.io/veda-docs/), [EIS](https://eis.smce.nasa.gov/about.html), and [MAAP](https://ops.maap-project.org), you can also keep using the resources provided by these projects. Through the use of open-source technology, we make sure our services are interoperable and exchangeable.

Reviewer: revert back to lowercase github in urls

Reviewer: in all the urls

## To Get US GHG Center JupyterHub access:

>  The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://forms.gle/4Z7rkUkeK9MW4joi7)

Reviewer: missing a period in the end

* Make sure you have a [GitHub Account](https://docs.GitHub.com/en/get-started/signing-up-for-GitHub/signing-up-for-a-new-GitHub-account). Take note of your [GitHub username](https://docs.GitHub.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-GitHub/managing-email-preferences/remembering-your-GitHub-username-or-email).
* Fill out the [request form](https://forms.gle/4Z7rkUkeK9MW4joi7) and provide needed information.
* Watch your email for notification of authorization and the invite to join the [US GHG Center Hub Access GitHub Team](https://GitHub.com/orgs/US-GHG-Center/teams/ghgc-hub-access).

* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your GitHub credentials.

## To access User Notebooks

This site provides [Jupyter notebooks](../user_data_notebooks/casagfed-carbonflux-monthgrid-v3_User_Notebook.ipynb) showing how to load and analyze Earth data in the interactive cloud computing environment.

> Further instructions are included in each notebook.

Reviewer: add a line break after "notebook."

this all would have been super easy if you had just accepted my suggestions directly

## To Get US GHG Center JupyterHub access:

 The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://docs.google.com/forms/d/e/1FAIpQLSdai8otCdrVQzJgev8mjDhzKyCg7jcrB3UeTXNHoCiaMKrkaQ/viewform).

* Make sure you have a [GitHub Account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). Take note of your [GitHub username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email).

> * Fill out the [request form](https://forms.gle/4Z7rkUkeK9MW4joi7) and provide needed information.

Reviewer: the url needs to be updated here too
